tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
-envlist = py27,py34
+envlist = py27,py310,py311
```

Comment:
I'm not very familiar with how this list is used, so feel free to ignore if I'm off base, but why does this not contain the same versions of Python that are used on Github Actions (3.7, 3.8, 3.9, 3.10, 3.11)? Would it make sense to replace `py27` with `py37,py38,py39`?

Reply:
it probably should. didn't notice the github actions the first time around (I think this is just used for local testing of changes, so you can say something like `tox -e py310` or whatever.